builtin/providers/google/resource_compute_instance.go

```diff
@@ -58,6 +58,12 @@ func resourceComputeInstance() *schema.Resource {
 							ForceNew: true,
 						},
 
+						"nvme": &schema.Schema{
```

Comment:
Hi.
I followed the same pattern used for definition of [`scratch disk type`](https://github.com/jusbrasil/terraform/blob/68115d4e2d25ba1086a3c0e16edca08029331dba/builtin/providers/google/resource_compute_instance.go#L55-L59) that the possible values for are either `"PERSISTENT"` or `"SCRATCH"` and it was defined as boolean with [`PERSISTENT` as default value](https://github.com/jusbrasil/terraform/blob/68115d4e2d25ba1086a3c0e16edca08029331dba/builtin/providers/google/resource_compute_instance.go#L388).
Should I change both?

About the unrelated files I removed them.